cucumber/authenticators_oidc/features/authn_oidc.feature

```diff
@@ -29,7 +29,10 @@ Feature: OIDC Authenticator - Hosts can authenticate with OIDC authenticator
         privilege: [ read, authenticate ]
         resource: !webservice
 
-    - !user alice
+    - !user
+      id: alice
+      annotations:
+       authn-oidc/keycloak: alice.somebody@cyberark.com
```

Comment:
I was thinking about the annotation more along the lines of:
```yml
- !user
      id: alice
      annotations:
       authn-oidc/identity: alice.somebody@cyberark.com
```
which would allow the user to be selected given a JWT with the subject `alice.somebody@cyberark.com`. This lines up with the way annotations are used for mapping in other authenticators: [Azure](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Operations/Services/azure_authn.htm#ConfiguretheAzureAuthenticator).